core-common/src/main/java/eu/arrowhead/common/dto/shared/IcmpPingResponseDTO.java

```diff
@@ -0,0 +1,68 @@
+package eu.arrowhead.common.dto.shared;
+
+import java.io.Serializable;
+
+public class IcmpPingResponseDTO implements Serializable{
+
+	//=================================================================================================
+	// members
+
+	private static final long serialVersionUID = 7382047276356365361L;
+
+	private Boolean successFlag;
```

Comment:
In a response object, every member where null is not possible should be primitive. successFlag and timeoutFlag can't be null for sure. In case of the others, I'm not so sure.

Reply:
fixed